spec/controllers/auth/registrations_controller_spec.rb

```diff
@@ -3,6 +3,45 @@
 RSpec.describe Auth::RegistrationsController, type: :controller do
   render_views
 
+  shared_examples 'checks for enabled registrations' do |path|
+    it 'redirects if it is in single user mode while it is open for registration' do
+      Fabricate(:account)
+      Setting.open_registrations = true
```

Comment:
Because `Setting` is global, changing it might affect other tests that run after, randomly. Storing the original value and restoring might be necessary